nanome_loaders/PPTReader/PPTReader.py

```diff
@@ -27,6 +38,10 @@ def is_open(self):
         return self.__plugin.menu == self.__menu
 
     def __del__(self):
+        try:
+            os.remove(self._tmp_dir)
+        except:
+            pass
         if self._base_name != "":
```

Comment:
aren't these files in the temp dir you just tried to delete?

Reply:
No, the _base_name files are going to be in the tmp directory.
The tmp_dir is a directory, also in tmp directory. I had to create it because soffice only output to directories